app/src/main/java/com/eveningoutpost/dexdrip/utils/ListActivityWithMenu.java

```diff
@@ -14,7 +14,7 @@
 /**
  * Created by stephenblack on 6/8/15.
  */
-public class ListActivityWithMenu extends ListActivity implements NavigationDrawerFragment.NavigationDrawerCallbacks {
+public abstract class ListActivityWithMenu extends ListActivity implements NavigationDrawerFragment.NavigationDrawerCallbacks {
```

Comment:
ah, I knew I was missing something here, I should spend some time to learn more about inheritance in java